src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
Lottery should not extend ColorSupplier, because **ColorSupplier is just a service which has some functionality**. <br/>You can just use ColorSupplier as a Lottery's field.
```suggestion
public class Lottery {
    private ColorSupplier colorSupplier = new ColorSupplier();
```